x-pack/test/api_integration/apis/ml/calculate_model_memory_limit.ts

```diff
@@ -100,7 +100,7 @@ export default ({ getService }: FtrProviderContext) => {
       },
     },
     {
-      testTitleSuffix: '4 influencers, split by customer_id and filtering by country code',
+      testTitleSuffix: '4 influencers, split by city and manufacturer + filtering by country code',
```

Comment:
Probably worth adding to the test title that it uses `2 detectors`

Reply:
Done in c9d8a00d30